docs/config.rst

```diff
@@ -46,7 +46,12 @@ core
 
     If ``true``, Gordon will continue running even if installed & configured plugins can not be loaded. Plugin exceptions will be logged as warnings with tracebacks.
 
-    If ``false``, Gordon will exit out if it can't load one or more plugins.
+    If ``false``, Gordon will exit out if it can't load one or more plugins
+
+.. option:: metrics='metrics-provider-name'
```

Comment:
It's convention to have the type rather than the "what" for the value, e.g. `.. option:: metrics="STR"` (also note double quotes).